packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -287,6 +287,8 @@ Media.updateTransceiver = ({meetingId, remoteQualityLevel}, peerConnection, tran
  * @returns {Promise.<MediaStream>}
  */
 Media.getDisplayMedia = (options, config = {}) => {
+  const resolutionConf = config.resolution || {};
+  const {screenResolution} = Config.meetings;
   const shareConstraints =
     options.sharePreferences && options.sharePreferences.shareConstraints ?
       options.sharePreferences.shareConstraints :
```

Comment:
The intention of shareConstraints was to provide the hight width and frame rate, But i agree that sometimes the user just care to send the hight and width and dont care for rest of the things